### ISSUE TITLE
Cannot find module node_modules/junit2json/dist/cjs/index.js

### ISSUE THREAD

**Issue by jamesnetherton** [Kesin11/ts-junit2json#313]
I use this project in a GitHub action. Since 3.1.8 was released it fails with:

```
Cannot find module node_modules/junit2json/dist/cjs/index.js
```

The `dist` directory seems to be duplicated:

```
$ find node_modules/junit2json
...
node_modules/junit2json/dist/dist/cjs/index.js
```

**Comment by Kesin11** [comment]
@jamesnetherton 
Oops. Thanks for the report. It looks like I broke it with a change in the release process. I will fix it right away.

**Comment by Kesin11** [comment]
@jamesnetherton 
I just released [v3.1.9](https://www.npmjs.com/package/junit2json/v/3.1.9) and mark v3.1.8 as deprecated.
Prease try it.

**Comment by jamesnetherton** [comment]
> Prease try it.

Looks good. Thanks @Kesin11!
